src/styles/GlobalStyle.ts

```diff
@@ -2,14 +2,46 @@ import styled, { createGlobalStyle } from 'styled-components';
 
 export const GlobalStyle = createGlobalStyle`
   * {
+    box-sizing: border-box;
+    margin: 0;
     color: ${props => props.theme.text_primary_color};
   }
+
   html {
     font-size: 62.5%;
   }
+  
   body {
+    font-size: 1.6rem;
     background-color: ${props => props.theme.background_color};
```

Comment:
p5;
기본 배경 테마를 `background_primary`로 하기로 했던 거 같습니다!
`background_color`는 오타였다고 회의 때 수영님이 말씀해주셨습니다.
```suggestion
    background-color: ${props => props.theme.background_primary};
```

Reply:
p4: 저도 이 부분은 다른 분들도 빠른 적용이 필요할 것 같아서 theme.ts 오타 같이 수정해주실 수 있을까요? 제 작업인데 죄송하게도 부탁드립니다..🙏

Reply:
추가적으로 `src/styles/theme.ts` 파일에서 [24번째 줄](https://github.com/1e5i-Shark/algobaro-fe/blob/70dad0b8c40811443d30caf88b06ef5326e5a048/src/styles/theme.ts#L24)에 다크 테마 때 기본 배경 변수명이 `background_color`로 되어 있는데 이것을 `background_primary`로 수정을 해주실 수 있으실까요?

모달 컴포넌트 구현 이슈보다 현재 이슈 task가 더 먼저 merge될 거 같기에 요청드립니다!

Reply:
오 그러네요!! `background_primary` 부분 수정하겠습니다~